src/features/venueMap/components/VenueTypeSection/VenueTypeSection.tsx

```diff
@@ -9,28 +9,37 @@ type Props = {
   venueTypeSelected: string | null
   venueTypeMapping: VenueTypeMapping
   onSelect: (venueTypeCode: VenueTypeCode) => void
+  venueCountByTypes?: Partial<VenuesCountByType>
 }
 
 export const VenueTypeSection: FunctionComponent<Props> = ({
   venueTypeSelected,
   venueTypeMapping,
   onSelect,
+  venueCountByTypes,
 }) => {
   return (
     <React.Fragment>
       <Spacer.Column numberOfSpaces={6} />
       <Typo.Title1>{venueTypeMapping.title}</Typo.Title1>
       <Spacer.Column numberOfSpaces={6} />
-      {Object.entries(venueTypeMapping.children).map(([venueTypeCode, label]) => (
-        <React.Fragment key={venueTypeCode}>
-          <RadioButton
-            label={label}
-            isSelected={venueTypeSelected === label}
-            onSelect={() => onSelect(venueTypeCode as VenueTypeCode)}
-          />
-          <Spacer.Column numberOfSpaces={6} />
-        </React.Fragment>
-      ))}
+      {Object.keys(venueTypeMapping.children).map((venueTypeCode) => {
+        const label = venueTypeMapping.children[venueTypeCode as VenueTypeCode] ?? ''
+        const isSelected = venueTypeSelected === label
+        const complement = venueCountByTypes?.[venueTypeCode as VenueTypeCode] ?? '0'
```

Comment:
```suggestion
        const complement = venueCountByTypes?.[venueTypeCode as VenueTypeCode] ?? 0
```
techniquement, vu que les autres valeurs sont aussi des numbers